Added script and templates, supporting the changing of colors in any term emu or color featured application.

ReFleXzZ opened the pull request:
Added support of color change for every terminal emulator or even tool, that has the possibility to theme as long as the template has the right keys.

Added support for:

- kitty
- ghostty
- wezterm
- alacrity
- foot

Additional tools with coloring support:

- Zellij

Miscellaneous changes:

Changed the environment variables in "applycolor.sh" to use "export", so child processes and scripts like "term_gen_colors.sh" inherit the variable values

ReFleXzZ committed:
Added script and templates for support of changing colors in any term emu.

Added support of color change for every terminal emulator or even tool,
that has the possibility to theme as long as the template has the right
keys.

Added support for:

- kitty
- ghostty
- wezterm
- alacrity
- foot

Additional tools with coloring support:

- Zellij

Miscellaneous changes:

Changed the environment variables in "applycolor.sh" to use "export", so
child processes and scripts like "term_gen_colors.sh" inherit the
variable values

LeVraiArdox commented on src/share/sleex/scripts/colors/term_gen_colors.sh:
```diff
@@ -0,0 +1,132 @@
+
+if [ -t 1 ]; then for s in "${seqs[@]}"; do printf "%b" "$s"; done; fi
+
+echo "Theme für ${TERMINAL} wurde erstellt: ${OUTPUT_FILE[${TERMINAL}]}"
```
Careful, german

ReFleXzZ replied:
Oops, dangerous 😀 I'm correcting some mistaktes. Hopefully I don't need to open a new pr? It's my first pull request ever :D

LeVraiArdox reviewed (commented):
Careful on the german echo.

Is it tested on every terminals ?

ReFleXzZ committed:
general fixes

- Fixed a Variable causing the "Force Color on Terminals" not work.
  - Fixed a german echo, translating it to english and moving it back to
the loop, right where it belongs. (Sometimes I switch languages on the
fly, sorry :$)

ReFleXzZ committed:
misc

LeVraiArdox commented:
Before I merge, have you tested on every terminals ?

ReFleXzZ commented:
Yes I did. But I did before alacritty completely moved to toml. So I need to fix the file ending for the config.

ReFleXzZ committed:
Minifix

changed the template file ending for alacritty, since alacritty moved
permanently to toml.
Replaced a comment, since it's contents were not correct.

ReFleXzZ committed:
Please don't ask... xD

I accidentally killed a row of numbers somewhen, so I've put them back
in their place now :D

ReFleXzZ commented:
Done :p

ReFleXzZ commented:
Only thing you might find not working proberly is the colors with wezterm, somehow some colors (even though, they're the same) are just plain white. But this is not because of the script, since this problem keeps existing even when only using the sequences being pushed through the pts.
